File: Problem 1/dennis.java
class EulerOne
{
    public static void main(String[] args)
    {
        int foo = 0;
        int bar = 0;
        int x = 0;
        int y = 0;

        for(int i = 1; x < 1000; i++)
        {
            x = i * 3;
            y = i * 5;
            if(x < 1000)
                foo += x;
            if(y < 1000)
                bar += y;
            if(x % 5 == 0)
                foo -= x;
        }
        System.out.println(foo + bar);
    }
}

File: Problem 3/dennis.cpp
#include <iostream>
#include <cmath>

long fermatiFormuoli(long a, long b);

long partA(long n) { return ((n+1)/2); };
long partB(long n) { return ((n-1)/2); };

int main()
{
    long n = 600851475143;
    long answer = 0;
    answer = fermatiFormuoli(partA(n), partB(n));
    std::cout << answer << '\n';

    return 0;
}

long fermatiFormuoli(long a, long b)
{
    long foo = a + b;
    long bar = a - b;

    std::cout << "a is " << a << '\n';
    std::cout << "b is " << b << '\n';

    std::cout << "a + b " <<  foo << std::endl;
    std::cout << "a - b " <<  bar << std::endl;
    return 0;
}
